src/components/KTooltip/KTooltip.vue

```diff
@@ -61,7 +62,7 @@ const props = defineProps({
   */
   placement: {
     type: String as PropType<PopPlacements>,
-    default: 'bottom',
+    default: 'top',
```

Comment:
question: why are we changing the default here?

Reply:
Oops, this was by accident, changing back.